Doc/library/typing.rst

```diff
@@ -1500,6 +1510,76 @@ These are not used in annotations. They are building blocks for declaring types.
    ``True`` as the value of the ``total`` argument. ``True`` is the default,
    and makes all items defined in the class body required.
 
+   It is possible for a ``TypedDict`` type to inherit from one or more TypedDict types
```

Comment:
```suggestion
   It is possible for a ``TypedDict`` type to inherit from one or more other ``TypedDict`` types
```

Reply:
We should probably also add that TypedDicts are not allowed to inherit from anything that is not a TypedDict, notably including `Generic`.